Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN    ln --symbolic --no-dereference --force /usr/share/zoneinfo/$TZ /etc/local
 RUN    apt update                                                                  \
     && apt upgrade --yes                                                           \
     && apt install --yes                                                           \
-        autoconf bison clang-6.0 cmake curl flex gcc libboost-test-dev             \
+        autoconf bison clang-6.0 clang-6.0++ cmake curl flex gcc libboost-test-dev \
```

Comment:
I am reasonably certain that not only does a clang-6.0++ package not exist, clang-6.0 installs clang++.

Reply:
Removed.